examples/bpk-component-modal-v2/stories.tsx

```diff
@@ -58,3 +58,56 @@ export const NoFullScreenOnMobileNoTitle = NoFullScreenOnMobileNoTitleExample;
 export const NoHeader = NoHeaderExample;
 export const MultipleModals = MultipleModalsExample;
 export const Contrast = ContrastExample;
+
+// Due to how iframes work we can pass a local url to load the stories above.
+// Attempted to use a Custom Iframe component with a react portal and ref to
+// render components but it didn't have the desired effect.
+const visualWrapper = (id: string, zoomEnabled: boolean = false) => (
```

Comment:
visualWrapper
```suggestion
const VisualWrapper = (id: string, zoomEnabled: boolean = false) => (
```

Then we can render it like a normal component: 


```jsx
<VisualWrapper id={'bpk-component-modal-v2--default'} />
```